packages/odate/odate.0.7/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "hugo.heuzard@gmail.com"
+authors: [ "Hugo Heuzard" ]
+license: "MIT"
+homepage: "https://github.com/hhugo/odate"
+bug-reports:  "https://github.com/hhugo/odate/issues"
+dev-repo: "git+https://github.com/hhugo/odate.git"
+
+synopsis: "Date & Duration Library"
+
+description: """
+Simple date and duration manipulation. Also implement duration printer
+based on string format. Already implemented in opalang
+[http://opalang.org/]. For documentation about the format, see :
+[http://doc.opalang.org/value/stdlib.core.date/Duration/try_generate_printer].
+"""
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "dune" {build}
```

Comment:
seems like it's missing lower bounds?
```suggestion
  "dune" {>= "2.2" & build}
```

Reply:
The `dune` dependencies should not be marked as `build`. I can't find the link to the issue that originally describes the reason why, but it is mentioned in the wiki: https://github.com/ocaml/opam-repository/wiki/PR-checks#dune-checks

Reply:
@hhugo I believe that `build` should not be there but a lower bound is necessary for Dune